.github/workflows/ci_cd.yml

```
@@ -65,7 +72,7 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: ${{ matrix.image }}
-          path: /tmp/${{ matrix.image }}.tar
+          path: 
```

I think this needs to stay the same :thinking: What was the reason for this change?

```suggestion
          path: /tmp/${{ matrix.image }}.tar
```

---

Oh, sorry. My mistake.

---

Fixed